app/controllers/users_controller.rb

```diff
@@ -1,5 +1,5 @@
 class UsersController < ApplicationController
-  before_action :authenticate_user!
+  load_and_authorize_resource except: :sign_out_and_redirect
 
```

Comment:
- [ ] Kindly solve this authorization error I am getting when I try to access the root page 

<img width="1081" alt="image" src="https://github.com/shayan1234554321/The-Knowledge-Hub/assets/79058364/e51b50f1-f7b6-46d8-bdb9-d74104ec640f">